src/h5web/toolbar/controls/DomainSlider/DomainSlider.tsx

```diff
@@ -53,9 +53,11 @@ function DomainSlider(props: Props) {
   const rootRef = useRef(null);
   const tooltipRef = useRef<DomainTooltipHandle>(null);
 
-  useClickAway(rootRef, () => {
-    toggleEditing(false);
-    tooltipRef.current?.cancelEditing();
+  useClickOutside(rootRef, () => {
+    if (isEditing) {
+      toggleEditing(false);
+      tooltipRef.current?.cancelEditing();
+    }
```

Comment:
Adding a condition here so we don't re-render the slider every time the user clicks around... I think React is smart enough to not re-render everything when the state doesn't change, but it still does a bit of work -- also, there's no point in cancelling editing in the tooltip anyways.